core/src/main/java/com/netease/arctic/catalog/CatalogLoader.java

```diff
@@ -156,4 +165,39 @@ private static ArcticCatalog buildCatalog(String impl) {
           String.format("Cannot initialize Catalog, %s does not implement Catalog.", impl), e);
     }
   }
+
+  private static void setAuthConfigsWithProperties(CatalogMeta catalogMeta, Map<String, String> props) {
+    boolean disableAmsAuth = PropertyUtil.propertyAsBoolean(props,
+        CatalogMetaProperties.AUTH_AMS_CONFIGS_DISABLE, CatalogMetaProperties.AUTH_AMS_CONFIGS_DISABLE_DEFAULT);
+
+    if (disableAmsAuth) {
```

Comment:
We support set authorization conifgs from properties no matter if allow load configs from ams.

Reply:
I recommend to move this logic into `Catalog::initialize`，So all catalogs could be supported no matter where it was created.